### PR TITLE
added detect-movement example project

### PR DESCRIPTION
**Pull request by edihasaj**
link from my github source  https://github.com/edihasaj/detect-moevement-lorawan

### CODE REVIEW

**Comment by edihasaj** [comment]
it is tested and it is working. Maybe you can contribute by adding an image in the readme section.

**Comment by epiller** [comment]
Hey @edihasaj, thanks for contributing! There are a few things that would need to be addressed first before merging this. I'll comment them in the code but it boils down to:  
There's no need to use the delay() function - in its current state, the example can take up to 200 seconds to react and send data after an event has happened. Instead, by adding an interrupt (or simply reading the PIR pin every second), you could make it send data only if an event happens. This would save battery life, improve response and send less payloads if there's no movement.

File: examples/Rapid Development Kit/detect-movement/detect-movement.ino [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  while((!debugSerial) && (millis()) < 10000){}         // Wait for the Serial monitor to be open (so you can see all output) and if it isn't open in 10 seconds, run the sketch
+  while (!modem.init()) { delay(1000); }                // Initialize the modem. If it fails, retry every 1 second until it succeeds
+  
+  pinMode(PIRSensPin, INPUT);                           // Read the status of our PIR sensor and save it in 'doorState' variable.
+}
+
+void loop() {
+  // put your main code here, to run repeatedly:
+  readSensors();
+  sendSensorValues();
```
Comment (epiller) [inline examples/Rapid Development Kit/detect-movement/detect-movement.ino:29]: This should be removed and called from the readSensors function instead

File: examples/Rapid Development Kit/detect-movement/detect-movement.ino [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  delay(sendEvery*1000);
```
Comment (epiller) [inline examples/Rapid Development Kit/detect-movement/detect-movement.ino]: This should be moved from here into readSensors function as well, but with a 1 second delay only

File: examples/Rapid Development Kit/detect-movement/detect-movement.ino [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+void readSensors() {                                    // Name of our function that we'll call from the loop function below
```
Comment (epiller) [inline examples/Rapid Development Kit/detect-movement/detect-movement.ino:32]: Would be cool if it sends data if something happens. Something like this:

```
pirValue = analogRead(PIRSensPin);
if (pirValue != previousPirValue) {
  if (pirValue) {
    debugSerial.println("Motion detected!");
  } else {
    debugSerial.println("Motion expired.");
  }
  sendSensorValues();
  previousPirValue = pirValue;
}
```

You'd just need to define a global variable ```previousPirValue```.

**Review by epiller — CHANGES_REQUESTED** [review]

**Review by edihasaj — COMMENTED** [review]
added changes, check if everything is as excepted. Thank you for your time.

**Pull request merged**